job/job.go

```diff
@@ -0,0 +1,188 @@
+package job
+
+import (
+	"bytes"
+	"compress/gzip"
+	"encoding/json"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3iface"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+	"time"
+)
+
+type Job struct {
+	Id string `json:"id"`
+}
```

Comment:
Do you need this type? It only contains the ID of the job, I wonder if it would be just simpler to pass and return a simple string called `jobID`, IMHO of course, maybe there was a reason for having this type 🤷‍♂

Reply:
I thought I did. That's the only value I care about: https://auth0.com/docs/users/guides/bulk-user-exports#sample-results

Reply:
sorry, probably you define this type because you're unmashalling the JSON into an instance of this type, that's why you have the type 👍